### DISCUSSION TITLE
In WinUI, how to implement splitter bar that can be dragged to resize different panes?

### DISCUSSION THREAD

**Discussion by gautambjain** [microsoft/microsoft-ui-xaml#8606] [Q&A] [answered]
In WinUI, how to implement splitter bar that can be dragged to resize different panes?

I am looking for a draggable splitter bar that separates 2 panes. When dragged, it would resize the pane accordingly.

Thanks.

**Comment by roxk** [comment, marked as answer]
You can use [GridSplitter](https://learn.microsoft.com/en-us/windows/communitytoolkit/controls/gridsplitter) in the toolkit.

**Comment by castorix** [comment]
There was a similar thread : https://github.com/microsoft/microsoft-ui-xaml/discussions/8158

**Comment by gautambjain** [comment]
Thanks @roxk  and @castorix 

:o)
